renovate.json

```diff
@@ -0,0 +1,13 @@
+{
+  "extends": [
+    "config:base",
+    ":preserveSemverRanges",
+    ":ignoreUnstable",
+    ":respectLatest",
```

Comment:
Minor question but I do not really understand the benefits of these lines by @kethinov.

Reply:
It's been a while since I researched these for other projects, but if I recall correctly we adopted the rules for the following reasons:

- `preserveSemverRanges`: Prevents it from rewriting your dependencies from `~` to `^` or vice versa so you can pin them to whatever degree of specificity you like.
- `ignoreUnstable`: Allows Renovate to bump beta/pre-release dependencies if you opt-in to any and will not bump them if the existing version is not already an unstable version.
- `respectLatest`: Similar to above. Will not upgrade to anything on npm that isn't tagged with "latest."